deployments/docker-compose/docker-compose.yaml

```diff
@@ -31,3 +35,13 @@ services:
     expose:
       - 27017
     restart: unless-stopped
+  minio:
+    image: 'minio/minio'
+    ports:
+      - 9000:9000
+    volumes:
+      - /etc/letsencrypt/live/${WEB_HOST}/fullchain.pem:/root/.minio/certs/public.crt
+      - /etc/letsencrypt/live/${WEB_HOST}/privkey.pem:/root/.minio/certs/private.key
+    environment:
+      - MINIO_ROOT_USER=minioadmin
```

Comment:
Let's use environment variables, for secrets, etc. .ENV file will be loaded by docker compose please see ${SERVER_SECRET}